### DISCUSSION TITLE
Is there a way to quickly switch between multiple Notes accounts?

### DISCUSSION THREAD

**Discussion by Parazyne** [standardnotes/forum#2637] [Suggestions] [answered]
**Describe the issue**
I have more than 1 Standard Notes account.  I often want to switch between them but to do so requires signing out then going through the complete sign in process for the desired account.

**To Reproduce**
1. Go to Account drop down
2. Select "Sign Out", then "Confirm"
3. Select "Sign In"
4. Enter Email, Password, 2FA if enabled

**Alternative behaviors**
1. On the Account drop down have a "Switch Account" selection.  That would invoke a list of accounts previously signed into.  Select an account and it immediately switches to the new account.  If it's already signed into that account then no entry of email/password is required.  If not then email/password are queried.  This alternative would obviously require Standard Notes to be signed into more than 1 account at a time.
2. On the Account drop down have a "Switch Account" selection.  That would invoke a list of accounts previously signed into.  Select an account and it automatically signs you out of the current one and queries for the new email/password.  This is similar to how Google account switching works.
3. Allow for multiple instances of Standard Notes to run at the same time.  For example, on Linux (on which I'm running) if one right clicks on many of the desktop icons there's a "New Window" option on the drop down. Selecting that option creates a whole new, separate and concurrently executing instance of the program.

**Comment by JaspalSuri** [comment]
Hi @Parazyne, thanks for the detailed feedback! I'll be sure to pass it along to the dev team. 🙂 

While the ability to switch accounts within the apps isn't currently supported, you can use the desktop and [web](https://app.standardnotes.org) apps on your computer simultaneously to easily manage multiple accounts. You can use the web app on your mobile device for small, quick edits as the UI currently isn't optimized for smaller displays.

**Comment by myreli** [comment, marked as answer]
This is now supported: https://standardnotes.com/help/82/can-i-sign-into-multiple-accounts-at-the-same-time
